docs/src/components/scripts.js

```diff
@@ -9,8 +9,12 @@ const Scripts = () => {
         dangerouslySetInnerHTML={{
           __html: `
           // Workaround: Make links on our Markdown files work on the static site
-          var anchors = document.querySelectorAll("#see-also + ul li a")
-          Array.prototype.slice.call(anchors).map(function(el) { el.href = el + "/index.html" })
+          var anchors = document.querySelectorAll(".sidebar a, .documentation a")
```

Comment:
Note: if the users has not javascript turned on (for some silly reason), then this will not work.

Reply:
(or if JS breaks or fails to load, as happens far more often than people assume it does)

Reply:
This script just formats the links on the docs site to allow offline navigation. If JS is turned off, functionality will remain the same but links will redirect to the folder containing the static html file, instead of the actual file. The static site does not need JS to function.